server/src/services/notification_service.ts

```diff
@@ -42,8 +42,9 @@ export declare type Notification = {
 }
 
 export async function sendNotification(notification: Notification) {
-  if(HASTIC_WEBHOOK_URL === null) {
-    throw new Error(`Can't send notification, HASTIC_WEBHOOK_URL is undefined`);
+  if(HASTIC_WEBHOOK_URL === null || HASTIC_WEBHOOK_URL === undefined) {
+    console.log(`HASTIC_WEBHOOK_URL not set, skip follow notification: ${notification.message}`);
```

Comment:
Maybe something like this?
```suggestion
    console.log(`HASTIC_WEBHOOK_URL is not set, skip sending notification: ${notification.message}`);
```